arm-sql/2014-04-01/swagger/connectionPolicies.json

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
Won't fix. Operations API is implemented in another file, https://github.com/Azure/azure-rest-api-specs/blob/master/arm-sql/2014-04-01/swagger/operations.json, so this is a false positive. We get this linter error in every file other then operations.json.